Sources/NIOChatServer/main.swift

```diff
@@ -139,7 +140,7 @@ defer {
 // First argument is the program path
 let arguments = CommandLine.arguments
 let arg1 = arguments.dropFirst().first
-let arg2 = arguments.dropFirst().dropFirst().first
+let arg2 = arguments.dropFirst(2).dropFirst().first
```

Comment:
Small one-off mistake? That drops 3 elements now but was previously dropping 2.

Should it be just:

```
arguments.dropFirst(2).first
```

?

Reply:
yikes, sorry

Reply:
Fixed!

Reply:
👍 :shipit: